pkg/address/address.go

```diff
@@ -59,3 +59,30 @@ func HexToAddress(hexAddr string) (b58 string, err error) {
 	b58 = base58.EncodeAlphabet(bytes, base58.BTCAlphabet)
 	return
 }
+
+// Returns an EIP55 Wanchain compliant hex string representation of the address.
+// See https://wandevs.org/docs/difference-between-wanchain-and-ethereum/
+// https://github.com/wanchain/go-wanchain/blob/b238c203ca7dc6a578d57c0c473bec0fcf2431bd/common/types.go#L173
+func EIP55ChecksumWanchain(address string) string {
+	v := []byte(Remove0x(strings.ToLower(address)))
+	sha := sha3.NewLegacyKeccak256()
+	_, err := sha.Write(v)
+	if err != nil {
+		logger.Error(err)
+	}
```

Comment:
Early return here?

Reply:
`""` or `err` ?

Reply:
In case if we handle we need also add err handling here https://github.com/trustwallet/blockatlas/blob/master/pkg/address/address.go#L27

Reply:
I think we can just return original string if there is an error, same for `EIP55Checksum`

Reply:
Left as in `EIP55Checksum`

Reply:
I would definitely return original string after logging in this error branch.  If sha.Write() fails, who knows what will sha.Sum() return?  Maybe empty array --> then we will overrun, or maybe all zeroes --> we will generate wrong result.